src/libstore/local-store.cc

```diff
@@ -1035,7 +1057,11 @@ StorePath LocalStore::addToStoreFromDump(const string & dump, const string & nam
 {
     Hash h = hashString(hashAlgo, dump);
 
-    auto dstPath = makeFixedOutputPath(method, h, name);
+    auto dstPath = makeFixedOutputPath(name, FixedOutputInfo {
+        method,
+        h,
+        {},
```

Comment:
Can you not have references with addToStoreFromDump?

Reply:
@matthewbauer I would say all those new `{}` are things that probably deserve auditing :)